content/participants/ttjc@adithyakam/index.md

```diff
@@ -38,22 +38,36 @@ Can't Wait to fill this area!!! Stay tuned
 
 My projects:
 
-<!--
-#### binder: tinder for books
+#### Moviesss
 
-_description_ do you wish to meet people who have read the same book so that you can talk hours about the same book? Try this.
+_description_ : It is a kind of netflix clone,where user canserch for movies,trendingmovies and also add movies to there favorities.
```

Comment:
Hey! @adithyakam some small mistakes here:
* where user **can search** for movies, **trending movies** and also add movies to **their favorites**.